### PR TITLE
Add change passcode settings page

### PR DESCRIPTION
**Pull request by orhoj**
## Purpose
Make it possible for a user to change their passcode.

## Changes
- Added the change passcode page to settings.
- Made input fields #white instead of #off-white.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/browser-wallet/src/popup/pages/ChangePasscode/ChangePasscode.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                                label="Enter new passcode"
+                                rules={{
+                                    required: tSetup('setupPasscode.form.passcodeRequired'),
+                                    minLength: { value: 6, message: tSetup('setupPasscode.form.passcodeMinLength') },
+                                }}
+                            />
+                            <FormPassword
+                                control={f.control}
+                                name="newPasscodeRepeated"
+                                label="Enter new passcode again"
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/ChangePasscode/ChangePasscode.tsx:80]:
```suggestion
                                label="Confirm new passcode"
```

Comment (shjortConcordium) [reply packages/browser-wallet/src/popup/pages/ChangePasscode/ChangePasscode.tsx:80]: Also these labels should be i18n'ed.

File: packages/browser-wallet/src/popup/pages/ChangePasscode/ChangePasscode.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                    return (
+                        <>
+                            <FormPassword
+                                control={f.control}
+                                name="currentPasscode"
+                                label="Enter current passcode"
+                                className="m-b-20"
+                                rules={{
+                                    required: tSetup('setupPasscode.form.passcodeRequired'),
+                                    validate: validateCurrentPasscode(),
```
[the remark this answers is not in the document]
Comment (orhoj) [reply packages/browser-wallet/src/popup/pages/ChangePasscode/ChangePasscode.tsx:65]: Let's keep it for now.

File: packages/browser-wallet/src/popup/pages/ChangePasscode/i18n/en.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const t = {
+    changePasscode: 'Change passcode',
+    incorrectPasscode: 'Incorrect passcode',
+    labels: {
+        currentPasscode: 'Enter current passcode',
+        newPasscode: 'Enter new passcode',
+        newPasscodeRepeated: 'Enter new passcode again',
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/ChangePasscode/i18n/en.ts:7]:
```suggestion
        newPasscodeRepeated: 'Confirm new passcode',
```
Do you not agree with this linguistic improvement?

File: packages/browser-wallet/src/popup/pages/ChangePasscode/i18n/da.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import type en from './en';
+
+const t: typeof en = {
+    changePasscode: 'Opdater kodeord',
+    incorrectPasscode: 'Forkert kodeord',
+    labels: {
+        currentPasscode: 'Enter current passcode',
+        newPasscode: 'Enter new passcode',
+        newPasscodeRepeated: 'Enter new passcode again',
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/ChangePasscode/i18n/da.ts:9]:
```suggestion
        currentPasscode: 'Skriv nuværende adgangskode'
        newPasscode: 'Skriv ny adgangskode',
        newPasscodeRepeated: 'Gentag ny adgangskode',
```

Comment (shjortConcordium) [reply packages/browser-wallet/src/popup/pages/ChangePasscode/i18n/da.ts:9]: Er kodeord egentligt ikke lidt mærkeligt?

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**